tdrs-backend/tdpservice/search_indexes/parsers/tanf_parser.py

```diff
@@ -95,3 +97,15 @@ def parse(datafile):
         else:
             logger.warn("Parsing for type %s not yet implemented", record_type)
             continue
+
+def validate(family_case_schema, t1):
```

Comment:
can `family_case_schema` and `t1` be renamed to be more generic? i'm thinking `validate` can be reused to accept _any_ schema object and document type